src/components/Photo/DocumentLiveCapture.js

```diff
@@ -57,11 +60,11 @@ export default class DocumentLiveCapture extends Component<Props, State> {
       isPreviewCropped: true
     }
     this.props.onCapture(documentCapture)
-    this.setState({ isLoading: false })
+    this.setState({ isLoading: false, btnDisabled: false })
   }
 
   captureDocumentPhoto = () => {
-    this.setState({ isLoading: true })
+    this.setState({ isLoading: true, btnDisabled: true })
```

Comment:
I somehow think you could just use `isLoading` state without setting `btnDisabled` which eventually is the same...

Reply:
I agree. Would also better describe code-wise why the button is being disabled.